### DISCUSSION TITLE
Configuration to apply insecure patches

### DISCUSSION THREAD

**Discussion by gustavo-bonfim** [biomejs/biome-vscode#160] [Ideas]
It would be nice to have a setting to apply unsafe fixes like tailwind class ordering.

Or some configuration to change the category of a rule (safe or unsafe)

**Comment by ematipico** [comment]
How would this confirmation look like @GustavoBonfimS ?

**Comment by gustavo-bonfim** [comment]
```json
  "biome.applyUnsafe": true,
```

Or something similar, I think that would be enough to apply unsafe fixes.

As I also mentioned, it would be interesting to have the option to change the classification of the rules, but it might be a little more complex.

**Reply by ematipico** [reply]
With classification you mean to change a safe fix to unsafe a vice versa? 

That won't work. The reason why we have unsafe fixes is because they might break your code, due to the nature of some dynamic patterns inside the language.

If we make fixes dynamic based on code and/or configuration, we break people's expectations.

It's the analyzer's duty to understand if a fix is safe or not, not the user. 

I know you're asking this because we want to apply class sorting as a safe fix ;)

**Reply by gustavo-bonfim** [reply]
Got it, it makes sense. 😄
Thanks for the feedback. 🙏🏻
